CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ### [Unreleased]
 
+## [17.5.4] - 2024-07-23
+### Fix for Jacksons SingleArgumentConstructorPojo issue
```

Comment:
Tiny thing - It might be good to title this so
### Fixed
- Jackson will now accept single argument constructors for pojos without the need for an annotation